sparkler-api/src/main/java/edu/usc/irds/sparkler/Constants.java

```diff
@@ -140,17 +140,23 @@ public static SparklerConfiguration newDefaultConfig(){
 
     interface solr { // Solr Fields
         String ID = "id";
-        String JOBID = "jobId";
+        String CRAWL_ID = "crawl_id";
         String URL = "url";
         String GROUP = "group";
-        String LAST_FETCHED_AT = "lastFetchedAt";
-        String NUM_TRIES = "numTries";
+        String FETCH_TIMESTAMP = "fetch_timestamp";
+        String RETRIES_SINCE_FETCH = "retries_since_fetch";
         String NUM_FETCHES = "numFetches";
-        String DEPTH = "depth";
+        String CRAWLER_DISCOVER_DEPTH = "crawler_discover_depth";
         String SCORE = "score";
         String STATUS = "status";
-        String LAST_UPDATED_AT = "lastUpdatedAt";
-        String PLAIN_TEXT = "plainText";
+        String LAST_UPDATED_AT = "last_updated_at";
+        String EXTRACTED_TEXT = "extracted_text";
+        String CONTENT_TYPE = "content_type";
+        String FETCH_STATUS_CODE = "fetch_status_code";
+        String SIGNATURE = "signature";
+        String OUTLINKS = "outlinks";
+        String CRAWLER_FETCH_DEPTH = "crawler_fetch_depth";
```

Comment:
Nope. I think that would mean something else. `FETCH_DEPTH` would be appropriate as it is the iteration number when the page is fetched.
This is not implemented yet. You may see this in next PR.